plasmapy/classes/species.py

```diff
@@ -237,7 +236,6 @@ def plot_time_trajectories(self, plot="xyz"):  # coveralls: ignore
         """
         from astropy.visualization import quantity_support
         import matplotlib.pyplot as plt
-        from mpl_toolkits.mplot3d import Axes3D
```

Comment:
An oversight on my part that I didn't automate testing visualization, my bad. But this will break the plots unless we keep the `import` or move it to the beginning of the file - I wasn't sure we should be doing that as it'll be only necessary for visualization, but that's probably way too premature optimization.

Pep8 won't catch that breaking either, matplotlib is weird like that on the 3d plotting end.

Reply:
OK, I'll re-add it.